protocol/communities/community.go

```diff
@@ -409,152 +398,56 @@ func (o *Community) MarshalJSONWithMediaServer(mediaServer *server.MediaServer)
 			communityItem.CommunityAdminSettings.PinMessageAllMembersEnabled = o.config.CommunityDescription.AdminSettings.PinMessageAllMembersEnabled
 		}
 	}
-	return json.Marshal(communityItem)
+	return communityItem, nil
 }
 
 func (o *Community) MarshalJSON() ([]byte, error) {
```

Comment:
I think the end goal would be to no longer use `MarshalJSON` and only use `MarshalJSONWithMediaServer`, though maybe we still use `MarshalJSON` for the DB?

Reply:
I was wondering if I should update all methods that return `communities.Community' to return `[]json.RawMessage' in api.go?
* RequestCommunityInfoFromMailserver
* RequestCommunityInfoFromMailserverWithShard
* FetchCommunity
* JoinedCommunities

Reply:
ah, never mind, I saw your previous comment :) 
I'll update all the methods and get rid of MarshalJSON.